src/scripts/main.js

```diff
@@ -1,3 +1,168 @@
 'use strict';
+// import { pushNotification } from './notification.js';
 
-// write code here
+// Sort table
+
+const sortButtons = document.querySelectorAll('thead tr th');
+const tbody = document.querySelector('tbody');
+const rows = Array.from(document.querySelectorAll('tbody tr'));
+
+const sortState = new Array(sortButtons.length).fill(false);
+
+sortButtons.forEach((button, index) => {
+  button.addEventListener('click', () => {
+    const isDescending = sortState[index];
+
+    const sortedRows = rows.sort((rowA, rowB) => {
+      const cellA = rowA.children[index].innerHTML;
+      const cellB = rowB.children[index].innerHTML;
+
+      if (
+        button.innerHTML === 'Name' ||
+        button.innerHTML === 'Position' ||
+        button.innerHTML === 'Office'
+      ) {
+        return isDescending
+          ? cellB.localeCompare(cellA)
+          : cellA.localeCompare(cellB);
+      } else if (button.innerHTML === 'Salary') {
+        const salaryA = parseFloat(cellA.replace(/[$,]/g, ''));
+        const salaryB = parseFloat(cellB.replace(/[$,]/g, ''));
+
+        return isDescending ? salaryB - salaryA : salaryA - salaryB;
+      } else {
+        return isDescending
+          ? parseFloat(cellB) - parseFloat(cellA)
+          : parseFloat(cellA) - parseFloat(cellB);
+      }
+    });
+
+    tbody.innerHTML = '';
+    sortedRows.forEach((row) => tbody.appendChild(row));
+
+    sortState[index] = !sortState[index];
+  });
+});
+
+// Selected row
+
+rows.forEach((row) => {
+  row.addEventListener('click', () => {
+    rows.forEach((r) => {
+      if (r !== row) {
+        r.classList.remove('active');
+      }
+    });
+
+    row.classList.toggle('active');
+  });
+});
+
+// notification
+const pushNotification = (posTop, posRight, title, description, type) => {
+  // Створюємо нове повідомлення
+  const notification = document.createElement('div');
+
+  document.body.append(notification);
+
+  notification.insertAdjacentHTML(
+    'afterbegin',
+    `
+    <h2 class="title">${title}</h2>
+    <p class="description">${description}</p>
+    `,
+  );
+
+  notification.setAttribute('class', `notification ${type}`);
+  notification.setAttribute('data-qa', `notification`);
+  notification.style.position = 'absolute';
+
+  notification.style.top = posTop + 'px';
+  notification.style.right = posRight + 'px';
+
+  setTimeout(() => {
+    notification.style.visibility = 'hidden';
+    notification.remove(); // Після зникнення видаляємо повідомлення з DOM
```

Comment:
```suggestion
    notification.remove();
```
it's clear from the code 🙂 Same above
But even if you add a comment be consistent - use English language everywhere)